src/main/java/org/wso2/carbon/securevault/aws/common/AWSVaultUtils.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.securevault.aws.common;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.securevault.aws.exception.AWSVaultException;
+
+import java.util.Properties;
+
+import static org.wso2.carbon.securevault.aws.common.AWSVaultConstants.LEGACY_PROPERTIES_PATH;
+import static org.wso2.carbon.securevault.aws.common.AWSVaultConstants.NOVEL_PROPERTIES_PATH;
+import static org.wso2.carbon.securevault.aws.common.AWSVaultConstants.REGEX;
+import static org.wso2.carbon.securevault.aws.common.AWSVaultConstants.SECRET_REPOSITORIES;
+
+/**
+ * Util methods used in the AWS Vault extension.
+ */
+public class AWSVaultUtils {
+
+    private AWSVaultUtils() {
+
+    }
+
+    private static final Log log = LogFactory.getLog(AWSVaultUtils.class);
+
+    /**
+     * Util method to get the properties based on legacy or novel method used for defining the property
+     * in the configurations file.
+     *
+     * @param properties   Configuration properties.
+     * @param propertyName Name of the required property.
+     * @return Property value.
+     */
+    public static String getProperty(Properties properties, String propertyName) {
```

Comment:
Add a newline after the method signature. Fix in other places too.

Reply:
Added. Added in other places too.